app/src/helper.py

```diff
@@ -23,6 +23,25 @@ def get_day_delta(resource_date):
         else:
             return from_datetime - from_datetime
 
+    # https://codereview.stackexchange.com/a/253174/234246
+    @staticmethod
+    def get_setting(settings, path, default=None):
+        result = settings
+        for key in path.split("."):
+            result = result.get(key)
+            if result is None:
+                return default
+        return result
+
+    @staticmethod
+    def get_whitelist(whitelist, path, default=[]):
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Default values in Python are created exactly once, when the function is defined. If that object is changed, like the default object in this example, subsequent calls to the function will refer to the changed object, leading to confusion. We recommend that you set the default value to `None`, then check inside the function if the parameter is `None` before creating the desired mutable object.

[Learn more](https://docs.python.org/3/faq/programming.html#why-are-default-values-shared-between-objects)